src/v/redpanda/admin/api-doc/transaction.json

```diff
@@ -92,6 +92,46 @@
                     ]
                 }
             ]
+        },
+        {
+            "path": "/v1/transaction/{group_id}/unsafe_abort_group_transaction",
```

Comment:
Nit: this looks a bit inconsistent, because group_id can be easily confused with transactional_id (when comparing with other endpoints)

Reply:
Thats a good point, I moved the {group_id} to the end of the path.